base/threadingconstructs.jl

```diff
@@ -95,6 +95,7 @@ function threading_run(fun, static)
     if !isempty(failed_tasks)
         throw(CompositeException(map(TaskFailedException, failed_tasks)))
     end
+    return map(Base.get_task_tls, tasks)
```

Comment:
How about simply:
```suggestion
    return tasks
```

Reply:
Is there other useful things inside the tasks?

Reply:
One thing that's nicer is that the show method for a vector of tasks is a lot tidier than a dict. So running a threaded for loop in the  repl would be tidier.